### PR TITLE
Feature/a dvantage support mo 6790

### PR DESCRIPTION
**Pull request by whcemyesil**

### CODE REVIEW

**Review by fabb — APPROVED** [review]

File: sdk/PrebidMobile.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -451,7 +451,7 @@
 					};
 					C2B16E1A1EE1BE69006304C0 = {
 						CreatedOnToolsVersion = 8.3.2;
-						DevelopmentTeam = HR94J6TSB3;
+						DevelopmentTeam = 282JR47M4L;
```
Comment (mathaeus) [inline sdk/PrebidMobile.xcodeproj/project.pbxproj:462]: Which team is that?

**Review by mathaeus — CHANGES_REQUESTED** [review]

File: sdk/PrebidMobile.xcodeproj/project.pbxproj (hunk 1 above)
Comment (mathaeus) [reply sdk/PrebidMobile.xcodeproj/project.pbxproj:462]: Is this even relevant?

Comment (whcemyesil) [reply sdk/PrebidMobile.xcodeproj/project.pbxproj]: screen shot 2018-02-14 at 11 59 06

Comment (whcemyesil) [reply sdk/PrebidMobile.xcodeproj/project.pbxproj:462]: willhaben Cem Yesil personal team

**Pull request merged**